Backend/dev/migrations/dynamic-offer-driver-app/0464-added-index-on-daily-stats-and-payout-order.sql

```diff
@@ -0,0 +1,3 @@
+CREATE INDEX idx_payout_status ON atlas_driver_offer_bpp.daily_stats USING btree (payout_status);
```

Comment:
We should not create indicies on low cardinal columns, else it could become very costly. Please remove this index.

> Low-cardinality refers to columns with few unique values.